.azure-pipelines-gh-pages.yml

```diff
@@ -7,7 +7,7 @@ trigger:
 
 jobs:
   - job: build_and_publish_docs
-    container: ccfciteam/ccf-ci:oe0.13.0-quote-ex
+    container: ccfciteam/ccf-ci:oe0.13.0-xxd
```

Comment:
Required to decode the hex-encoded quote and endorsements and replace the call to `perl` in `verify_quote.sh`.